internal/service/cloudbackupsnapshotexportbucket/data_source_cloud_backup_snapshot_export_bucket.go

```diff
@@ -14,8 +15,10 @@ func DataSource() *schema.Resource {
 		ReadContext: datasourceRead,
 		Schema: map[string]*schema.Schema{
 			"id": {
-				Type:     schema.TypeString,
-				Required: true,
+				Type:       schema.TypeString,
+				Optional:   true,
+				Computed:   true,
+				Deprecated: fmt.Sprintf(constant.DeprecationParamByVersion, "1.18.0") + " Will not be an input parameter, only computed.",
```

Comment:
do we need to update the documentation?

Reply:
knit: will be nice to create a new constant with the explanation included, sounds like the way to go in next deprecations

Reply:
In the documentation, the `id` was not mentioned 👍 only:

* `project_id` - (Required) The unique identifier of the project for the Atlas cluster.
* `export_bucket_id` - (Required) Unique identifier of the snapshot export bucket.